sycl/test-e2e/Matrix/common.hpp

```diff
@@ -5,8 +5,15 @@
 
 using bfloat16 = sycl::ext::oneapi::bfloat16;
 
-constexpr float BF16_EPSILON = 10e-2;
-constexpr float FLOAT_EPSILON = 10e-3;
+// The most of the time fails related to floating point calculations
```

Comment:
minor wording:
Most of the time, failures related to floating-point calculations (both float and bfloat16) are caused by accumulation errors rather than the algorithm itself.
If it is an algorithm issue, the calculated result gap from the reference would be much bigger. To avoid flaky test results while catching algorithm errors, we are increasing here the accuracy threshold.

Reply:
done